src/med_bench/estimation/dml.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+
+
+from med_bench.estimation.base import Estimator
+from med_bench.nuisances.propensities import estimate_treatment_probabilities
+from med_bench.nuisances.cross_conditional_outcome import estimate_cross_conditional_mean_outcome_nesting
+
+
+class DoubleMachineLearning(Estimator):
+    """Implementation of double machine learning
+
+    Parameters
+    ----------
+        settings (dict): dictionnary of parameters
+        lbda (float): regularization parameter
```

Comment:
```suggestion
        alpha (float): regularization parameter
``` Note: sklearn calls regularization  parameters `alpha`